src/components/Languages/LanguagesPresentation.js

```diff
@@ -44,23 +40,23 @@ function LanguagesPresentation({
             />
           </div>
           <div className="mt-5">
-            {Object.keys(allSitesList).map((parentLanguage) => {
+            {allSitesList.map((parentLanguage) => {
               // Generating class for border color
-              const borderColor = parentLanguagesData[parentLanguage]
-                ? `border-[${parentLanguagesData[parentLanguage]}]`
+              const borderColor = parentLanguage
+                ? `border-[${parentLanguage}]`
                 : 'border-gray'
```

Comment:
Oops, I think we over-reached when we took this part out. We could keep the parentLanguagesData (and the related parts of the LanguagesData and LanguagesContainer) and just change the key to parentLanguage.language here.